### ISSUE TITLE
My pattern is non-existant 

### ISSUE THREAD

**Issue by cheezy805** [mikebryant/ac-nh-turnip-prices#100]
Updated: prices were wrong. My bad. I literally wrote down 85/82 and still managed to enter 95/92. 

I started the week with a descending pattern that the Prophet said was fluctuating, but turns out it's a big spike. Gave me a weekly max of 133. Then my prices hit 174 and the Prophet just wigged out. I gambled and it did a big spike, reaching 435 currently. I'll update this at the end of the week. 

So far:
**First time buyer**: no
**Previous pattern**: descending
**Sunday price**: 95
**Prices so far**: 85, 82, 77, 87, 174, 435.

**Comment by Vankog** [comment]
follow-up to my pattern last week, I got some more data of that weir week:

![price outside predicted range](https://user-images.githubusercontent.com/3804909/79304486-c37a1f00-7ef1-11ea-9e0b-87a652fc2248.png)

While the thursday price now works, the friday morning price still fails.
It only works with 39+, so 5 points more than the game actually gave me.

Hope this helps!

**Comment by theRTC204** [comment]
> I started the week with a descending pattern that the Prophet said was fluctuating, but turns out it's a big spike. Gave me a weekly max of 133. Then my prices hit 174 and the Prophet just wigged out. I gambled and it did a big spike, reaching 435 currently. I'll update this at the end of the week.
> 
> So far:
> **First time buyer**: no
> **Previous pattern**: descending
> **Sunday price**: 95
> **Prices so far**: 95, 92, 77, 87, 174, 435.

Something's not quite right with those prices so far... According to the extracted code, there is only a single pattern that can possibly have it's initial phase be >= the buy price; that pattern is 0 (fluctuating)

Pattern 0, first price is based on `intceil(randfloat(0.9, 1.4) * basePrice)`
Pattern 1, first price is based on `intceil(randfloat(0.9, 0.85) * basePrice)`
Pattern 2, first price is based on `intceil((0.9 - randfloat(0, 0.05))* basePrice)`
Pattern 3, first price is based on `intceil(randfloat(0.9, 0.4) * basePrice)`

So, if your buy price was 95, and your first sell price was also 95, that means based on the code that was extracted from the game the only possible pattern for you would have been pattern 0.

Basically what I'm saying is - either the code that Ninji extracted is flawed, or your numbers are wrong.

**Comment by cheezy805** [comment]
> Basically what I'm saying is - either the code that Ninji extracted is flawed, or your numbers are wrong.

My numbers were wrong. Ignore me. Updated original post.
